modules/users/server/controllers/users/users.password.server.controller.js

```diff
@@ -138,6 +138,10 @@ exports.reset = function(req, res, next) {
 									message: errorHandler.getErrorMessage(err)
 								});
 							} else {
+							    // Remove sensitive data before login
+							    user.password = undefined;
```

Comment:
@lirantal When a user signs into the website, it also strips the password and salt. 
https://github.com/meanjs/mean/blob/0.4.0/modules/users/server/controllers/users/users.authentication.server.controller.js#L64
I am not sure why it needs to be stripped, but if we want to keep it consistent, then removing both the salt and the password would be appropriate. (If I had to take a guess, I don't think it matters because req.login is a method added to req by passport. And when passport performs a login, it already strips the sensitive info.)